code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -906,7 +906,7 @@
 
 /datum/reagent/toxin/leadacetate
 	name = "Lead Acetate"
-	description = "Used hundreds of years ago as a sweetener, before before it was realized it was made with poisonious lead."
+	description = "Used hundreds of years ago as a sweetener, before it was realized it was made with poisonious lead."
```

Comment:
They knew it was made with lead, just not that it was poisonous right?
Also *poisonous, made the same mistake

Reply:
actually "before they realized it was poisonous" is probably your best bet
I'm not tiered you're tired